player.gd

```diff
@@ -75,6 +77,12 @@ func play_animation():
 		_animated_sprite.play("IDLE")
 	elif movement_state == MovementState.JUMPING:
 		_animated_sprite.play("JUMPING")
+		if input.down[1]:
+			_animated_sprite.play("CROUCH_JUMP")
+		if velocity.y > 0:
+			_animated_sprite.frame = 3
```

Comment:
Can you create a separate animated sprite for the case of Falling vs Rising instead of specific frames of the jumping animation?